README.md

```diff
@@ -98,7 +98,9 @@ The format of this glossary is modeled after [HugoGiraudel/SJSJ](https://github.
 * **Conventional current**<a name="conventional-current"></a>:
     mental model envisioning current as running from positive to negative in a circuit.
 * Continuity:
-* cylon.js:
+* <a name="conventional-current"></a>**Conventional current**:
+  Mental model of current as running from positive to negative in a circuit
+* **[cylon.js](glossary/CYLONJS.md)**: JavaScript framework for robotics, physical computing, and the Internet of Things.
```

Comment:
Looks like you've added back the second `conventional current` line that I just killed. Could you delete the line starting with `<a name="conventional-current">`? Sorry for that confusion.

Your def looks good. Could you put a line break after colon (:) and indent the definition iself by four spaces (see other definitions in the page for examples)? This helps with scanning, I think...